### DISCUSSION TITLE
gnome-keyring sshfs

### DISCUSSION THREAD

**Discussion by Lichtmacher** [ltsp/ltsp#419] [Q&A]
gnome-keyring doesn't work on a sshfs filesystem.

Is that still an issue?
     /usr/share/ltsp/client/login/pamltsp

         # https://bugzilla.gnome.org/show_bug.cgi?id=730587
        rm -f /usr/bin/gnome-keyring-daemon

Some user could use gnome-keyring. For example skype decided to store user,pw there. Without you have to retype it at every start of skype.

**Comment by alkisg** [comment]
Hi, yes, the gnome-keyring developers haven't fixed that issue yet, see #22.

To fix it, contact the gnome-keyring developers in https://bugzilla.gnome.org/show_bug.cgi?id=730587.
To work around it in LTSP, use NFS home.

**Comment by Lichtmacher** [comment]
You post sadly many Burgreports on which the respective maintainers do not work.

**Comment by alkisg** [comment]
I think you're misunderstanding some open source concepts...

The gnome developers and maintainers work just fine. For some of them it's even their main job; they're paid to work on GNOME every day.
But they are not paid to solve your or my problems. Run `man gpl` and read the big "NO WARRANTY" section.
IF it affects their paying customers, then they might work on these issues.
IF you send patches or you pay some developer to send patches for you, then they may accept the fix.
IF many people complain about some issue, they MIGHT think it's important enough to fix it.

Check which of these tasks you did, to help in solving the issue that affects you. Are you a paying GNOME customer? Did you send a patch? Did you hire a developer to send a patch? Did you at least contact them?

Complaining about bugs of OTHER software, here in the LTSP discussions, will really NOT help in solving these bugs.

Of course, LTSP users do want to hear about bugs that might affect them, and about workarounds like the NFS home that I proposed, but don't expect to solve them by complaining here...

**Reply by Lichtmacher** [reply]
Sorry, I didn't want to complain here about the not working "others".
I am very thankfull for linux, for LTSP and the whole opensource scene.

I just wanted to give a bit of compassion to you.

Several times I googled a problem, realized it is a bug, and I found comments of "alksig" to it.
And sometimes  a comment like this "it doesn't work for many people if you leave it like this".
But of course it is theirs decision not just to include the fix, you provided.

**Reply by alkisg** [reply]
Thanks for clearing this up and sorry that I misunderstood. Cheers! :)

**Comment by Lichtmacher** [comment]
Just one addition to it:
I'm not a programer. I'm an administrator. So my role is to mediate between user and programer. I push open source by helping the users and finding workarounds for them. So I'm a bit sensitive to these little things -> It could nearly work, but there are just lacking 3 lines of code.
Also I appreciate the benefits of being able to fit a lot of things, that can't be done with closed source.

By the way. Is there a way to support LTSP?

**Comment by alkisg** [comment]
In open source, sometimes roles are blurry... and I have to be a teacher, sysadmin, programmer, bug reporter / triager or even bounty sponsor etc, depending on the issue at hand. It takes a lot more effort than I'd like it to, but the result is usually worth it.

About supporting LTSP, thank you for the thought; at this point it's not necessary.
Personally I'm available for hired help on LTSP related tasks to anyone that needs more than e.g. 5-10 minutes of free assistance, but I don't rely on that for a living.

Maybe you could ask other SSHFS users and see if you all can organize a small bounty for the solution to that gnome-keyring issue! (in GNOME side, not in LTSP side) :smile:
